MoreLinq.Test/SkipLastTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2016 Leandro F. Vieira (leandromoh). All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+using NUnit.Framework;
+using System.Linq;
+
+namespace MoreLinq.Test
+{
+    [TestFixture]
+    public class SkipLastTest
+    {
+        [Test]
+        public void SkipLastWithNullSequence()
+        {
+            Assert.ThrowsArgumentNullException("source", () => MoreEnumerable.SkipLast<int>(null, 1));
+        }
+
+        [TestCase(1, 5, 0)]
+        [TestCase(1, 5, -1)]
+        public void SkipLastWithCountLesserThanOne(int start, int count, int skip)
```

Comment:
Test name?

`start` and `count` seem constant across all cases here so no need to parameterise on those.